supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorInitializationActor.scala

```diff
@@ -0,0 +1,74 @@
+package cromwell.backend.impl.htcondor
+
+import akka.actor.Props
+import cromwell.backend.BackendLifecycleActor.WorkflowAbortResponse
+import cromwell.backend.impl.htcondor.HtCondorInitializationActor._
+import cromwell.backend.validation.RuntimeAttributesKeys._
+import cromwell.backend.validation.Validation._
+import cromwell.backend.{BackendConfigurationDescriptor, BackendWorkflowDescriptor, BackendWorkflowInitializationActor}
+import wdl4s.util.TryUtil
+import wdl4s.{Call, ThrowableWithErrors, WdlExpression}
+
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+import scalaz.Scalaz._
+
+object HtCondorInitializationActor {
+  val FailOnStderrDefaultValue = true
+
+  def props(workflowDescriptor: BackendWorkflowDescriptor, calls: Seq[Call], configurationDescriptor: BackendConfigurationDescriptor): Props =
+    Props(new HtCondorInitializationActor(workflowDescriptor, calls, configurationDescriptor))
+}
+
+class HtCondorInitializationActor(override val workflowDescriptor: BackendWorkflowDescriptor,
+                                  override val calls: Seq[Call],
+                                  override val configurationDescriptor: BackendConfigurationDescriptor) extends BackendWorkflowInitializationActor {
+  /**
+    * Abort all initializations.
+    */
+  override def abortInitialization: Future[WorkflowAbortResponse] = ???
+
+  /**
+    * Validate that this WorkflowBackendActor can run all of the calls that it's been assigned
+    */
+  override def validate: Future[Unit] = {
+    //TODO: Expressions need to be resolved. EvalFunctions is needed. TBD in the PR.
+    //TODO: refactor old code to use validation functions from Validation object in Backend.
+    //TODO: Workflow options may need to be validated for JES.
+    //TODO: Move specifics from Jes to JesBackendInitializationActor.
+    Future {
+      TryUtil.sequence(calls map {
```

Comment:
The current structure also doesn't say which call failed validation.  I think the following addresses these problems, though I'm sure it can be improved:

```scala
  override def validate: Future[Unit] = {
    case class ErrorMessageAggregatedException(override val exceptionContext: String,
                                               override val errorMessages: Traversable[String]) extends MessageAggregation
    Future {
      val callsWithValidations = calls map { call => call -> validateRuntimeAttributes(call) }
      val failedValidations = callsWithValidations collect { case (call, validation) if validation.isFailure =>
        ErrorMessageAggregatedException(s"Runtime attribute validation failed for task '${call.taskFqn}': ", validation.left.toList) }
      
      if (failedValidations.nonEmpty) {
        throw new exception.AggregatedException("Runtime attribute validation failed", failedValidations)
      }
    }
  }

  private def validateRuntimeAttributes(call: Call): ErrorOr[(Option[String], Boolean)] = {
    val taskAttributes: Map[String, WdlExpression] = call.task.runtimeAttributes.attrs
    val docker = validateDocker(taskAttributes.get(Docker), () => "Failed to get Docker mandatory key from runtime attributes".failureNel)
    val failOnStderr = validateFailOnStderr(taskAttributes.get(FailOnStderr), () => FailOnStderrDefaultValue.successNel)
    (docker |@| failOnStderr) { (_, _) }
  }
```